dali/operators/image/paste/multipaste.h

```diff
@@ -172,6 +179,52 @@ class MultiPasteOp : public Operator<Backend> {
     }
   }
 
+  using Operator<Backend>::RunImpl;
+
+  void RunImpl(workspace_t<Backend> &ws) override {
+    const auto input_type_id = ws.template InputRef<Backend>(0).type().id();
+    TYPE_SWITCH(input_type_id, type2id, InputType, (uint8_t, int16_t, int32_t, float), (
```

Comment:
Can we make the supported types a define and list them only once in the file?

Reply:
Done (that and more).